gc.c

```diff
@@ -1957,6 +1957,12 @@ heap_page_free(rb_objspace_t *objspace, struct heap_page *page)
     free(page);
 }
 
+static inline bool
+heap_page_full_p(struct heap_page *page)
```

Comment:
shouldn't this be "empty" ?

Reply:
you are, of course, absolutely correct. Fixed now.